spec/models/hyrax/collection_type_spec.rb

```diff
@@ -1,4 +1,4 @@
-RSpec.describe Hyrax::CollectionType, type: :model do
+RSpec.describe Hyrax::CollectionType, clean_repo: true, type: :model do
```

Comment:
Did you add this to address remnants that cause problems with repeated test runs?